app/Community/Actions/LoadThinActivePlayersListAction.php

```diff
@@ -55,7 +55,7 @@ function () use ($lookbackMinutes, $minimumPermissions) {
                         'username' => $player->username,
                         'display_name' => $player->display_name,
                         'rich_presence' => $player->rich_presence,
-                        'game_title' => $player->game?->Title ?? '',
+                        'game_title' => $player->game->Title ?? '',
```

Comment:
In general, what error are these changes trying to address?

There's a `with(['game'])` clause on the query, but what if that returns a null game? Isn't `?->` safer?